app/lib/humane_transport_app.dart

```diff
@@ -12,6 +12,7 @@ class HumaneTransportApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      debugShowCheckedModeBanner: false,
```

Comment:
Does this remove the little red "DEBUG" tag in the corner of the screen?

Reply:
yes